airflow/utils/dag_processing.py

```diff
@@ -577,7 +566,7 @@ def __init__(self,
 
         self._last_zombie_query_time = None
         # Last time that the DAG dir was traversed to look for files
-        self.last_dag_dir_refresh_time = timezone.utcnow()
+        self.last_dag_dir_refresh_time = timezone.make_aware(datetime.fromtimestamp(0))
```

Comment:
Why this change?

Reply:
We want the files to be read during the first loop cycle.
https://github.com/apache/airflow/blob/master/airflow/utils/dag_processing.py#L717-L718